Exiled.API/Features/Roles/Scp096Role.cs

```diff
@@ -51,58 +59,45 @@ internal Scp096Role(Scp096GameRole baseRole)
         public HumeShieldModuleBase HumeShieldModule { get; }
 
         /// <summary>
-        /// Gets a value indicating SCP-096's ability state.
-        /// </summary>
-        public Scp096AbilityState AbilityState => Internal.StateController.AbilityState;
-
-        /// <summary>
-        /// Gets a value indicating SCP-096's rage state.
-        /// </summary>
-        public Scp096RageState RageState => Internal.StateController.RageState;
-
-        /// <summary>
-        /// Gets a value indicating whether or not SCP-096 can receive targets.
+        /// Gets .
         /// </summary>
-        public bool CanReceiveTargets => SubroutineModule.TryGetSubroutine(out Scp096RageCycleAbility ability) && ability._targetsTracker.CanReceiveTargets;
+        public Scp096RageCycleAbility Scp096RageCycleAbility { get; }
 
         /// <summary>
-        /// Gets a value indicating whether or not SCP-096 is currently enraged.
+        /// Gets .
         /// </summary>
-        public bool IsEnraged => RageState == Scp096RageState.Enraged;
+        public Scp096RageManager Scp096RageManager { get; }
 
         /// <summary>
-        /// Gets a value indicating whether or not SCP-096 is currently docile.
+        /// Gets .
         /// </summary>
-        public bool IsDocile => !IsEnraged;
+        public Scp096TargetsTracker Scp096TargetsTracker { get; }
```

Comment:
```suggestion
        public Scp096TargetsTracker TargetsTracker { get; }
```
Change the name of this property to the new one everytime it gets called above and below.